server/controllers/authController.api.js

```diff
@@ -39,3 +50,27 @@ exports.login = (req, res) => {
         }
     });
 }
+
+exports.userById = (req, res, next, id) => {
+    User.findById(id).exec((err, user) => {
```

Comment:
let's get the user and projects together

Reply:
okay I can move the populate to this method then

Reply:
You didn't populate in this method yet